tests/integration/main.rs

```diff
@@ -270,7 +270,9 @@ async fn test_get_collection_by_property() -> Result<(), DatastorersError> {
     );
 
     // Compare the two int arrays to validate that all inserted items have been fetched
-    assert_eq!(fetched_int_props.sort(), int_props.sort());
+    fetched_int_props.sort_unstable(); // Sort in place
+    int_props.sort_unstable(); // Sort in place
+    assert_eq!(fetched_int_props, int_props);
```

Comment:
This one was fun! Actually comparing unit to unit so always succeeded, since `.sort` is sort in place and returns unit `()`

Reply:
sneaky!